src/definitions/behaviors/form.js

```diff
@@ -705,7 +708,60 @@ $.fn.form = function(parameters) {
                       values[name] = false;
                     }
                   }
-                  else {
+                  else if(isCalendar) {
+                    // We must determine which item is the calendar module
+                    var $calendar = $parent.is(selector.uiCalendar)
```

Comment:
If using the above mentioned suggestion, this code can be reduced to `$calendar = $calendar[0] || false;`

Reply:
With the simplification, I think that this piece of code is now useless...